test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/CodeGenerationTests.cs

```diff
@@ -7,6 +7,7 @@
 using System.Reflection;
 using Microsoft.AspNetCore.Razor.Evolution;
 using Microsoft.AspNetCore.Razor.TagHelpers;
+using Microsoft.AspNetCore.Razor.Test.Common;
```

Comment:
Don't put anything in this namespace. Just put these types in the same namespaces as the product-related functionality.